<!DOCTYPE html>
<html>
<head>
  <title>TSCAN: Tools for Single-Cell ANalysis</title>
  <meta charset="utf-8">
  <meta name="description" content="TSCAN: Tools for Single-Cell ANalysis">
  <meta name="author" content="Zhicheng Ji, Hongkai Ji">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <aside class="gdbar">
    <img src="assets/img/bloomberg_shield.png">
  </aside>
  <hgroup class="auto-fadein">
    <h1>TSCAN: Tools for Single-Cell ANalysis</h1>
    <h2>A Noval Method for Single-Cell RNA-seq Pseudo Time Ordering</h2>
    <p>Zhicheng Ji, Hongkai Ji<br/></p>
  </hgroup>
    <a href="https://github.com/zji90/TSCAN-presentation/zipball/gh-pages" class="example">
     Download
    </a>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Single-cell RNA-seq is a powerful technology to investigate the heterogeneity of gene expressions on cell level, which is otherwise hard to detect by bulk RNA-seq.</p>

<p><center><img src="fig/singlecell_RNA-Seq_workflow.jpg" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Single-cell RNA-seq can avoid potential Simpson&#39;s paradox and increase temporal resolution:</p>

<p><center><img src="fig/Simpson's_paradox.png" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Many single-cell RNA-seq data involve continuous biological processes. </p>

<p>In differentiation process, some cells differentiate faster than the others even though they are collected at the same time point.</p>

<p>An artificial order of cells need to be constructed to reflect the gradual change of the transcriptome among different cells.</p>

<p>Monocle: unsupervised approach to construct pseudo time cell order using minimum-spanning-tree.</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/Monocle_demo.png" width="60%"></center></p>

<p>Source: Trapnell, C., et al. (2014). The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nature biotechnology.</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Disadvantages of current methods:</p>

<ul>
<li><p>Unsupervised methods cannot incorporate prior biological knowledge (e.g. expressions of marker genes).</p></li>
<li><p>Users have limited options to tune the pseudo time orderings.</p></li>
<li><p>No quantitative methods to evaluate different pseudo time orderings.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>TSCAN, a novel method &amp; software package:</p>

<ul>
<li><p>Can switch from unsupervised to supervised strategy to incorporate prior biological knowledge.</p></li>
<li><p>Plenty of options for users to conveniently tune the pseudo time orderings.</p></li>
<li><p>POS: quantitative methods to evaluate different pseudo time orderings.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>TSCAN Methods</h2>
  </hgroup>
  <article data-timings="">
    <ol>
<li>Perform principal component analysis.</li>
<li>Model-based clustering.</li>
<li>Connect all cluster centers with minimum-spanning-tree.</li>
<li>Project all points on the backbone path and form pseudo time ordering.</li>
</ol>

<p><center><img src="fig/TSCAN_overview.png" width="50%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>POS</h2>
  </hgroup>
  <article data-timings="">
    <p>Pseudo time Ordering Score: quantitatively measure different pseudo time orderings.</p>

<p>Requires prior knowledge of at least two cell sub-populations.</p>

<p>Cells collected at later time points should be assigned to later positions on pseudo time axis.</p>

<p>Definition for two cell sub-population case:</p>

<p>\(\frac{\sum_{i,j:\pi(i) \prec \pi(j)} (T_j-T_i)}{N_1*N_2}\)</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>Results</h2>
  </hgroup>
  <article data-timings="">
    <p>Testing datasets:</p>

<ul>
<li><p>Human skeletal muscle myoblasts: 4 time points; 271 cells; 47192 genes; marker: CKM</p></li>
<li><p>Mouse LPS stimulation: 5 time points; 363 cells; 27723 genes; marker: BCL3</p></li>
<li><p>Mouse ES-MEF: 2 time pints; 92 cells; 21711 genes; marker: Sox2</p></li>
</ul>

<p>Methods compared:</p>

<ul>
<li><p>Monocle</p></li>
<li><p>Unsupervised TSCAN</p></li>
<li><p>Supervised TSCAN (with marker gene information)</p></li>
<li><p>Marker gene</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>POS</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/POS.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>HSMM CDK1 gene</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/HSMM_CDK1.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>LPS STAT1 gene</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/LPS_STAT1.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-13" style="background:;">
  <hgroup>
    <h2>ESMEF Pou5f1 gene</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/ESMEF_Pou5f1.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-14" style="background:;">
  <hgroup>
    <h2>Differential gene analysis</h2>
  </hgroup>
  <article data-timings="">
    <p>Key GO terms:</p>

<p>HSMM: skeletal muscle cell differentiation (GO:0035914)</p>

<p>LPS: immune response (GO:0006955)</p>

<p>ES-MEF: embryo development (GO:0009790)</p>

<p><center><img src="fig/pooled_GO.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>Single time point</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/Singletimepoint.png" width="60%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>Software Package</h2>
  </hgroup>
  <article data-timings="">
    <p>TSCAN is already available on Bioconductor:</p>

<p><a href="http://bioconductor.org/packages/release/bioc/html/TSCAN.html">http://bioconductor.org/packages/release/bioc/html/TSCAN.html</a></p>

<p>TSCAN GUI can be directly launched on:</p>

<p><a href="https://zhiji.shinyapps.io/TSCAN">https://zhiji.shinyapps.io/TSCAN</a></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>Demo</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-18" style="background:;">
  <hgroup>
    <h2>Thank you!</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title='Introductions'>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Introductions'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Introductions'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Introductions'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Introductions'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Introductions'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='TSCAN Methods'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='POS'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Results'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='POS'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='HSMM CDK1 gene'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='LPS STAT1 gene'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='ESMEF Pou5f1 gene'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='Differential gene analysis'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='Single time point'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='Software Package'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='Demo'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='Thank you!'>
         18
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>
